src/seedu/addressbook/data/person/Email.java

```diff
@@ -11,7 +11,9 @@
     public static final String EXAMPLE = "valid@e.mail";
     public static final String MESSAGE_EMAIL_CONSTRAINTS =
             "Person emails should be 2 alphanumeric/period strings separated by '@'";
-    public static final String EMAIL_VALIDATION_REGEX = "[\\w\\.]+@[\\w\\.]+";
+    public static final String EMAIL_VALIDATION_REGEX0 = "\\S+@\\S+\\.com";
+    public static final String EMAIL_VALIDATION_REGEX1 = "\\S+@\\S+\\.org";
+    public static final String EMAIL_VALIDATION_REGEX2 = "\\S+@\\S+\\.edu.sg";
```

Comment:
Oh! I don't mean to say it is wrong. The best way to handle is to update the documentation and make your scope very clear to the user.